bokeh/embed.py

```diff
@@ -59,23 +87,40 @@ def components(plot_objects, resources=None):
             components({"Plot 1": plot_1, "Plot 2": plot_2})
             #=> (script, {"Plot 1": plot_1_div, "Plot 2": plot_2_div})
```

Comment:
I think `plot1_div`, etc. would be a little easier on the eyes. Would recommend that throughout (below for `plot2_dict` too, etc.)